c_src/sqlite3_nif.c

```diff
@@ -219,6 +220,12 @@ exqlite_open(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
         return make_error_tuple(env, "database_open_failed");
     }
 
+    conn->mutex = enif_mutex_create("exqlite:connection");
+    if (conn->mutex == NULL) {
+        enif_release_resource(conn);
+        return make_error_tuple(env, "failed to create mutex");
```

Comment:
Since this gets turned into an atom. `{:error, :failed_to_create_mutex}`

```suggestion
        return make_error_tuple(env, "failed_to_create_mutex");
```

Reply:
fixed